portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,19 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addFortune() {
+  const fortunes =
+      ['You will meet someone of importance to your future today', 
+            'Make sure to speak your truth today.', 
+            'The planets are totally alligned against you today', 
```

Comment:
aligned?

Reply:
My spelling skills -- second to none. Thank you for checking :)